### DISCUSSION TITLE
Create a work account

### DISCUSSION THREAD

**Discussion by HYAR7E** [vercel/community#501] [Help] [answered]
Hello there, i've land into a new job a few days ago and need to create a new vercel account and link it to my new github account (new accounts with the email the company gave me).

When trying to register with my new email i received the confirmation code in my phone and when i used it, it said the number is already taken (by my personal vercel account) and shows me an error, so i went to my personal account to try and unlink my phone from my personal account but i didn't find the option.

So while writing this i wanted to include a screenshot of the error but turns out my new work account was successfully created.

And that's all, this was going to be a "help me" post but the problem solved itselft, idk if this behaviour is intended so i'm publishing this anyway. Cheers :D

**Comment by samcx** [comment, marked as answer]
Can you send us a case at support@vercel.com so we can help resolve this for you? It will be better to resolve this via Vercel Support so we don't expose your work email here.
